cmd/nerdctl/container_inspect_linux_test.go

```diff
@@ -154,3 +155,54 @@ func TestContainerInspectContainsLabel(t *testing.T) {
 	assert.Equal(base.T, "foo", lbs["foo"])
 	assert.Equal(base.T, "bar", lbs["bar"])
 }
+
+func TestContainerInspectState(t *testing.T) {
+	t.Parallel()
+	testContainer := testutil.Identifier(t)
+	base := testutil.NewBase(t)
+
+	type testCase struct {
+		name, containerName, cmd string
+		want                     dockercompat.ContainerState
+	}
+	errStatus := ""
+	if base.Target == testutil.Docker {
+		errStatus = "created"
```

Comment:
Why differs for Docker?

Reply:
nerdctl : run error means task is nil so the `Status` is empty because `Status` comes from Task.
docker : run error gives => `Status` = `created`.

Reply:
Please add that in the code comment lines

Reply:
@AkihiroSuda Fixed